src/Sylius/Component/Channel/Resolver/ShippableCountriesResolverInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Channel\Resolver;
+
+use Sylius\Component\Core\Model\ChannelInterface;
+
+interface ShippableCountriesResolverInterface
+{
+    /**
+     * @param ChannelInterface|null $channel
+     *
+     * @return array
+     */
+    public function getShippableCountries(ChannelInterface $channel = null): array;
```

Comment:
`?ChannelInterface $channel = null` if it's nullable.

Btw. should it accept nulls at all?

Reply:
If it's null it should use the current channel to get the countries.